packages/components/forms/src/base-checkbox/GhostCheckbox.styles.ts

```diff
@@ -13,31 +13,31 @@ const getBaseGhostStyles = ({ isDisabled }): CSSObject => ({
   display: 'inline-flex',
   height: tokens.spacingM,
   justifyContent: 'center',
+  marginRight: tokens.spacingXs,
   width: tokens.spacingM,
 });
 
-const getCheckboxStyles = ({ isChecked, isDisabled, isIndeterminate }) => {
+const getCheckboxStyles = ({ isDisabled }) => {
   const baseStyle = {
     ...getBaseGhostStyles({ isDisabled }),
     backgroundColor: !isDisabled ? tokens.colorWhite : tokens.gray300,
     borderRadius: tokens.borderRadiusSmall,
     '& svg': {
       fill: !isDisabled ? tokens.colorWhite : tokens.gray300,
     },
-  };
-
-  const checkedStyle = {
-    backgroundColor: !isDisabled ? tokens.blue600 : tokens.gray300,
-    borderColor: !isDisabled ? tokens.blue600 : tokens.gray300,
-    '& svg': {
-      fill: !isDisabled ? tokens.colorWhite : tokens.gray600,
+    'input:indeterminate + &, input:checked + &': {
```

Comment:
This change was needed so it updates the GhostCheckbox correctly if the component is uncontrolloed.